data/json/items/tools.json

```diff
@@ -1098,6 +1098,52 @@
     "use_action": "HOTPLATE"
   },
   {
+    "id": "trangia_stove",
+    "sub": "hotplate",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "yellow",
+    "name": "spirit burner",
+    "description": "This is a small brass spirit burner with a screw-top lid and simmer ring, designed for cooking. It burns concentrated alcohol.",
+    "price" : 1000,
+    "material" : "copper",
+    "weight": 113,
+    "volume": 0,
+    "bashing": 1,
+    "cutting": 0,
+    "to_hit": 0,
+    "max_charges": 14,
+    "initial_charges": 7,
+    "charges_per_use": 1,
+    "turns_per_charge": 0,
+    "ammo": "conc_alcohol",
+    "revert_to": "null",
+    "use_action": "HOTPLATE"
+  },
+  {
+    "id": "popcan_stove",
+    "sub": "hotplate",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "light_gray",
+    "name": "soda can stove kit",
+    "description": "This is an ultra-light alcohol-burning stove with simmer ring, crafted by hand from a couple of aluminum soda cans. It comes with a 500ml plastic bottle to hold concentrated alcohol fuel.",
+    "price" : 250,
+    "material" : "aluminum",
+    "weight": 45,
+    "volume": 1,
```

Comment:
Same here: volume 1 is less than volume 2 of the bottle it has a component. It should have volume 3 at the very least.